recipes/CommonVoice/common_voice_prepare.py

```diff
@@ -15,6 +15,8 @@
 import torchaudio
 import unicodedata
 from tqdm.contrib import tzip
+import datasets
```

Comment:
I don't like the idea of introducing `datasets`. First, a user may not want to use `datasets` at all while now, it will force the user to use `datasets` because it is now a mandatory package (not external). Second, is it really necessary ? I know it can be easier to use `datasets` when dealing with a lot of datasets, but it introduces a new dependency which is unwanted at all cost. 

I'd like to get the opinions of @TParcollet and @mravanelli, but for me, I strongly disagree on introducing `datasets` for now. I'd prefer to wait a little bit more to see how we can introduce `datasets` in SpeechBrain with maybe a wrapper around it.

Reply:
I change the download dataset function and remove the dependencies to dataset. Please check download_locale function is common_voice_prepare.py